src/software/simulation/physics/physics_robot.cpp

```diff
@@ -371,6 +371,25 @@ void PhysicsRobot::setPosition(const Point& position)
     post_physics_step_functions.emplace(func);
 }
 
+void PhysicsRobot::setPositionOrientation(const Point& position, const Angle& angle)
```

Comment:
Couldn't we just do setOrientation here?

Reply:
Good call, I'll make that change!